optional_components/embeddings/app/config.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+HUGGINGFACE_MODEL = os.environ.get("HUGGINGFACE_MODEL", "thenlper/gte-large")
```

Comment:
Oh so if we pass when running
```
env: 
  - HUGGINGFACE_MODEL: "someother/model"
```
we can switch up which model we use!

Reply:
Yes. As long as it is an embedding model.